# BioFung
About The Project: BioFung allows fungi specific annotation for function and insight into pathways. It a database that allows comprehensive information on the metabolism of fungal species. This database is described in-depth within the manuscript and file for execution can be found at github NeeluBegum/BioFung.

Installation: BioFung has been placed on nextflow docker simple usuage and can be 

Independent Installation: BioFung operates with HMMER software and can be found on http://hmmer.org/ installation using the bash command line but can be used on a cluster or remotely. 

Usage: The BioFung database comes with scan.hmm.sh file for bash command and wrapper for the command line. The BioFung database "BioFung.hmm" requires unzipping (file size: 1.41GB). The input data requires protein fasta data. the output includes list of protein-encoding gene, evalue, percentage matching and corresponding identification of KEGG orthologs. This text file then can be used for further analysis and metabolism mapping using the KEGG database (https://www.genome.jp/kegg/pathway.html).

Contributing: Created by Begum N. based on KEGG annotations Contact: neelu.begum@kcl.ac.uk

Acknowledgements: KEGG, HMMER

Please checkout main group page for updated code: sysbiomelab/BioFung

check out BioFung on https://github.com/sysbiomelab for information and you can download BioFung from https://www.microbiomeatlas.org/downloads.php

Find out more about our research team on: https://www.sysbiomelab.com/
and my co-lab: https://www.kcl.ac.uk/research/functional-microbiome-group-moyes-group
